components/org.wso2.carbon.identity.oauth.stub/src/main/resources/OAuthAdminService.wsdl

```diff
@@ -395,7 +395,6 @@
                 <xs:sequence>
                     <xs:element minOccurs="0" name="OAuthVersion" nillable="true" type="xs:string"/>
                     <xs:element maxOccurs="unbounded" minOccurs="0" name="accessTokenClaims" nillable="true" type="xs:string"/>
-                    <xs:element minOccurs="0" name="accessTokenClaimsSeparationEnabled" type="xs:boolean"/>
```

Comment:
Is this added very recently? We can remove such properties from the wsdl after adding

Reply:
Introduced with @shashimalcse feature. Removing this as we do not want to keep this variable in the access token data structure anymore.